pennylane/transforms/decompositions/single_qubit_unitary.py

```diff
@@ -46,53 +45,6 @@ def _convert_to_su2(U, return_global_phase=False):
     return (U_SU2, exp_angles) if return_global_phase else U_SU2
 
 
-def zyz_decomposition(U, wire):
-    r"""Recover the decomposition of a single-qubit matrix :math:`U` in terms of
-    elementary operations.
-
-    Diagonal operations can be converted to a single :class:`.RZ` gate, while non-diagonal
-    operations will be converted to a :class:`.Rot` gate that implements the original operation
-    up to a global phase in the form :math:`RZ(\omega) RY(\theta) RZ(\phi)`.
-
-    .. warning::
-
-        When used with ``jax.jit``, all unitaries will be converted to :class:`.Rot` gates,
-        including those that are diagonal.
-
-    Args:
-        U (tensor): A 2 x 2 unitary matrix.
-        wire (Union[Wires, Sequence[int] or int]): The wire on which to apply the operation.
-
-    Returns:
-        list[qml.Operation]: A ``Rot`` gate on the specified wire that implements ``U``
-        up to a global phase, or an equivalent ``RZ`` gate if ``U`` is diagonal.
-
-    **Example**
-
-    Suppose we would like to apply the following unitary operation:
-
-    .. code-block:: python3
-
-        U = np.array([
-            [-0.28829348-0.78829734j,  0.30364367+0.45085995j],
-            [ 0.53396245-0.10177564j,  0.76279558-0.35024096j]
-        ])
-
-    For PennyLane devices that cannot natively implement ``QubitUnitary``, we
-    can instead recover a ``Rot`` gate that implements the same operation, up
-    to a global phase:
-
-    >>> decomp = zyz_decomposition(U, 0)
-    >>> decomp
-    [Rot(-0.24209529417800013, 1.14938178234275, 1.7330581433950871, wires=[0])]
-    """
-    warnings.warn(
-        "The zyz_decomposition function is deprecated and will be removed soon. Use"
-        " one_qubit_decomposition() with the keyword rotations='ZYZ'"
-    )
-    return _zyz_decomposition_old(U=U, wire=wire)
-
-
 def _zyz_decomposition_old(U, wire):
```

Comment:
if we're keeping this, perhaps we shouldn't keep the `_old` suffix?

Reply:
renamed to `_rot_decomposition`.